reviews/beautify_text.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+
+def beautify(text):
+    mapping = {'A': '𝔄', 'B': '𝔅', 'C': 'ℭ', 'D': '𝔇', 'E': '𝔈', 'F': '𝔉', 'G': '𝔊', 'H': 'ℌ', 'I': 'ℑ', 'J': '𝔍', 'K': '𝔎', 'L': '𝔏', 'M': '𝔐', 'N': '𝔑', 'O': '𝔒', 'P': '𝔓', 'Q': '𝔔', 'R': 'ℜ', 'S': '𝔖', 'T': '𝔗', 'U': '𝔘', 'V': '𝔙', 'W': '𝔚', 'X': '𝔛', 'Y': '𝔜', 'Z': 'ℨ', 'a': '𝔞', 'b': '𝔟', 'c': '𝔠', 'd': '𝔡', 'e': '𝔢', 'f': '𝔣', 'g': '𝔤', 'h': '𝔥', 'i': '𝔦', 'j': '𝔧', 'k': '𝔨', 'l': '𝔩', 'm': '𝔪', 'n': '𝔫', 'o': '𝔬', 'p': '𝔭', 'q': '𝔮', 'r': '𝔯', 's': '𝔰', 't': '𝔱', 'u': '𝔲', 'v': '𝔳', 'w': '𝔴', 'x': '𝔵', 'y': '𝔶', 'z': '𝔷'}
+    return ''.join(mapping.get(char, char) for char in text)
+
+
+def convert(text): return beautify(text)
+
+
+def main(text): return convert(text)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Load text from command line argument and beautify it.")
+    parser.add_argument('--text', help='Ugly text.')
+    args = parser.parse_args()
+    result = main(args.text)
+    print(f"Received text: {result}")
```

Comment:
actually this script as it is is useless because the console (at least the mine) does not support such characters and refuses to display them
would be better to store the results in some file

Reply:
Repeating `useless` twice in one code review can damage relations with colleague.
Script works fine in UTF-8 systems (windows is to be blamed here).

I would suggest something like: "The current code doesn't support Windows with their non-utf-8 encodings. Could you please fix it for me to be able test it properly? Many thanks"